filebeat/tests/system/test_modules.py

```diff
@@ -276,11 +281,6 @@ def clean_keys(obj):
         if "event.end" not in obj:
             delete_key(obj, "@timestamp")
 
-    # Remove event.ingested from testing, as it will never be the same.
-    if obj["event.dataset"] == "microsoft.defender_atp":
-        delete_key(obj, "event.ingested")
-        delete_key(obj, "@timestamp")
-
```

Comment:
can we drop the gsuite stuff below this too?

Reply:
done, had to expand the module into `dataset`s since that's what the stripping code works on, below was just a blanket module deletion.